### PR TITLE
fix(Dropdown) Return selected value to the label

### LINKED ISSUE #21887: aria-labelledby on Dropdown is set to Dropdown itself if no id provided
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
https://codesandbox.io/s/tv5h0?module=/example.js&file=/example.js

#### Actual behavior:

aria-labelledby is set to id of dropdown button and macOS voiceover does not read text, insteand says button

#### Expected behavior:
 instead should ignore and read text from span in dropdown

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by jurokapsiar**
Following change reoved selected value from the Dropdown label:

Fix Dropdown not annoucing placeholder/default value with VoiceOver on macOS @aubreyquinn (https://github.com/microsoft/fluentui/pull/22173)

This PR fixes the original issue by pointing aria-labelledby to the trigger button content.

Empty state:
image

Selected item:
image

Fixes #21887

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/66c282c7aa6ac773c8ec355fc990cbc234d4ee3a'>66c282c7aa6ac773c8ec355fc990cbc234d4ee3a</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
<details><summary>Perf tests with no regressions</summary>

Scenario | Current PR Ticks | Baseline Ticks | Ratio
:--- | ---:| ---:| ---:
AccordionMinimalPerf.default | [120](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AccordionMinimalPerf.default.html) | [108](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AccordionMinimalPerf.default_base.html) | 1.11:1
AttachmentMinimalPerf.default | [126](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AttachmentMinimalPerf.default.html) | [120](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AttachmentMinimalPerf.default_base.html) | 1.05:1
ListNestedPerf.default | [477](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListNestedPerf.default.html) | [454](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListNestedPerf.default_base.html) | 1.05:1
TableMinimalPerf.default | [343](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TableMinimalPerf.default.html) | [326](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TableMinimalPerf.default_base.html) | 1.05:1
TreeWith60ListItems.default | [139](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TreeWith60ListItems.default.html) | [132](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TreeWith60ListItems.default_base.html) | 1.05:1
CarouselMinimalPerf.default | [404](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CarouselMinimalPerf.default.html) | [389](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CarouselMinimalPerf.default_base.html) | 1.04:1
DropdownMinimalPerf.default | [2689](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DropdownMinimalPerf.default.html) | [2575](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DropdownMinimalPerf.default_base.html) | 1.04:1
VideoMinimalPerf.default | [554](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/VideoMinimalPerf.default.html) | [535](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/VideoMinimalPerf.default_base.html) | 1.04:1
AnimationMinimalPerf.default | [469](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AnimationMinimalPerf.default.html) | [455](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AnimationMinimalPerf.default_base.html) | 1.03:1
AttachmentSlotsPerf.default | [897](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AttachmentSlotsPerf.default.html) | [875](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AttachmentSlotsPerf.default_base.html) | 1.03:1
FormMinimalPerf.default | [344](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/FormMinimalPerf.default.html) | [334](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/FormMinimalPerf.default_base.html) | 1.03:1
BoxMinimalPerf.default | [274](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/BoxMinimalPerf.default.html) | [268](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/BoxMinimalPerf.default_base.html) | 1.02:1
ButtonMinimalPerf.default | [137](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ButtonMinimalPerf.default.html) | [134](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ButtonMinimalPerf.default_base.html) | 1.02:1
DialogMinimalPerf.default | [645](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DialogMinimalPerf.default.html) | [635](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DialogMinimalPerf.default_base.html) | 1.02:1
DividerMinimalPerf.default | [288](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DividerMinimalPerf.default.html) | [283](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DividerMinimalPerf.default_base.html) | 1.02:1
EmbedMinimalPerf.default | [3518](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/EmbedMinimalPerf.default.html) | [3462](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/EmbedMinimalPerf.default_base.html) | 1.02:1
HeaderMinimalPerf.default | [297](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/HeaderMinimalPerf.default.html) | [291](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/HeaderMinimalPerf.default_base.html) | 1.02:1
LabelMinimalPerf.default | [322](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/LabelMinimalPerf.default.html) | [317](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/LabelMinimalPerf.default_base.html) | 1.02:1
MenuMinimalPerf.default | [725](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/MenuMinimalPerf.default.html) | [713](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/MenuMinimalPerf.default_base.html) | 1.02:1
PopupMinimalPerf.default | [534](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/PopupMinimalPerf.default.html) | [526](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/PopupMinimalPerf.default_base.html) | 1.02:1
ReactionMinimalPerf.default | [308](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ReactionMinimalPerf.default.html) | [303](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ReactionMinimalPerf.default_base.html) | 1.02:1
RefMinimalPerf.default | [194](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/RefMinimalPerf.default.html) | [190](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/RefMinimalPerf.default_base.html) | 1.02:1
SkeletonMinimalPerf.default | [286](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SkeletonMinimalPerf.default.html) | [281](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SkeletonMinimalPerf.default_base.html) | 1.02:1
StatusMinimalPerf.default | [561](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/StatusMinimalPerf.default.html) | [548](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/StatusMinimalPerf.default_base.html) | 1.02:1
IconMinimalPerf.default | [504](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/IconMinimalPerf.default.html) | [495](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/IconMinimalPerf.default_base.html) | 1.02:1
AvatarMinimalPerf.default | [154](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AvatarMinimalPerf.default.html) | [152](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AvatarMinimalPerf.default_base.html) | 1.01:1
ChatDuplicateMessagesPerf.default | [247](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ChatDuplicateMessagesPerf.default.html) | [244](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ChatDuplicateMessagesPerf.default_base.html) | 1.01:1
ChatMinimalPerf.default | [618](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ChatMinimalPerf.default.html) | [613](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ChatMinimalPerf.default_base.html) | 1.01:1
CheckboxMinimalPerf.default | [2256](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CheckboxMinimalPerf.default.html) | [2239](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CheckboxMinimalPerf.default_base.html) | 1.01:1
DatepickerMinimalPerf.default | [4917](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DatepickerMinimalPerf.default.html) | [4846](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DatepickerMinimalPerf.default_base.html) | 1.01:1
DropdownManyItemsPerf.default | [567](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DropdownManyItemsPerf.default.html) | [559](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/DropdownManyItemsPerf.default_base.html) | 1.01:1
InputMinimalPerf.default | [1097](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/InputMinimalPerf.default.html) | [1089](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/InputMinimalPerf.default_base.html) | 1.01:1
ProviderMinimalPerf.default | [338](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ProviderMinimalPerf.default.html) | [335](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ProviderMinimalPerf.default_base.html) | 1.01:1
SplitButtonMinimalPerf.default | [3727](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SplitButtonMinimalPerf.default.html) | [3692](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SplitButtonMinimalPerf.default_base.html) | 1.01:1
TextMinimalPerf.default | [284](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TextMinimalPerf.default.html) | [280](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TextMinimalPerf.default_base.html) | 1.01:1
ToolbarMinimalPerf.default | [801](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ToolbarMinimalPerf.default.html) | [796](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ToolbarMinimalPerf.default_base.html) | 1.01:1
TreeMinimalPerf.default | [684](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TreeMinimalPerf.default.html) | [678](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TreeMinimalPerf.default_base.html) | 1.01:1
AlertMinimalPerf.default | [213](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AlertMinimalPerf.default.html) | [213](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/AlertMinimalPerf.default_base.html) | 1:1
HeaderSlotsPerf.default | [632](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/HeaderSlotsPerf.default.html) | [632](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/HeaderSlotsPerf.default_base.html) | 1:1
ListWith60ListItems.default | [511](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListWith60ListItems.default.html) | [513](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListWith60ListItems.default_base.html) | 1:1
LoaderMinimalPerf.default | [573](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/LoaderMinimalPerf.default.html) | [574](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/LoaderMinimalPerf.default_base.html) | 1:1
MenuButtonMinimalPerf.default | [1430](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/MenuButtonMinimalPerf.default.html) | [1433](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/MenuButtonMinimalPerf.default_base.html) | 1:1
ProviderMergeThemesPerf.default | [1066](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ProviderMergeThemesPerf.default.html) | [1063](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ProviderMergeThemesPerf.default_base.html) | 1:1
SliderMinimalPerf.default | [1437](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SliderMinimalPerf.default.html) | [1431](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SliderMinimalPerf.default_base.html) | 1:1
TableManyItemsPerf.default | [1614](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TableManyItemsPerf.default.html) | [1615](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TableManyItemsPerf.default_base.html) | 1:1
CustomToolbarPrototype.default | [2317](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CustomToolbarPrototype.default.html) | [2314](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CustomToolbarPrototype.default_base.html) | 1:1
TooltipMinimalPerf.default | [969](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TooltipMinimalPerf.default.html) | [965](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TooltipMinimalPerf.default_base.html) | 1:1
ButtonOverridesMissPerf.default | [1247](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ButtonOverridesMissPerf.default.html) | [1260](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ButtonOverridesMissPerf.default_base.html) | 0.99:1
CardMinimalPerf.default | [452](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CardMinimalPerf.default.html) | [458](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/CardMinimalPerf.default_base.html) | 0.99:1
GridMinimalPerf.default | [274](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/GridMinimalPerf.default.html) | [278](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/GridMinimalPerf.default_base.html) | 0.99:1
ImageMinimalPerf.default | [303](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ImageMinimalPerf.default.html) | [306](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ImageMinimalPerf.default_base.html) | 0.99:1
LayoutMinimalPerf.default | [289](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/LayoutMinimalPerf.default.html) | [291](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/LayoutMinimalPerf.default_base.html) | 0.99:1
PortalMinimalPerf.default | [138](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/PortalMinimalPerf.default.html) | [139](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/PortalMinimalPerf.default_base.html) | 0.99:1
RadioGroupMinimalPerf.default | [369](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/RadioGroupMinimalPerf.default.html) | [372](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/RadioGroupMinimalPerf.default_base.html) | 0.99:1
TextAreaMinimalPerf.default | [402](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TextAreaMinimalPerf.default.html) | [406](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/TextAreaMinimalPerf.default_base.html) | 0.99:1
ItemLayoutMinimalPerf.default | [971](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ItemLayoutMinimalPerf.default.html) | [988](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ItemLayoutMinimalPerf.default_base.html) | 0.98:1
ListCommonPerf.default | [522](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListCommonPerf.default.html) | [535](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListCommonPerf.default_base.html) | 0.98:1
SegmentMinimalPerf.default | [280](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SegmentMinimalPerf.default.html) | [286](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/SegmentMinimalPerf.default_base.html) | 0.98:1
FlexMinimalPerf.default | [223](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/FlexMinimalPerf.default.html) | [230](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/FlexMinimalPerf.default_base.html) | 0.97:1
ListMinimalPerf.default | [414](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListMinimalPerf.default.html) | [428](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ListMinimalPerf.default_base.html) | 0.97:1
ButtonSlotsPerf.default | [443](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ButtonSlotsPerf.default.html) | [467](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ButtonSlotsPerf.default_base.html) | 0.95:1
ChatWithPopoverPerf.default | [313](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ChatWithPopoverPerf.default.html) | [330](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/ChatWithPopoverPerf.default_base.html) | 0.95:1
RosterPerf.default | [895](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/RosterPerf.default.html) | [948](https://fluentuipr.z22.web.core.windows.net/pull/23407/perf-test-northstar/RosterPerf.default_base.html) | 0.94:1

</details><div id="perfComment9424"/>

**Review by YuanboXue-Amber — APPROVED** [review]

File: packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -573,7 +573,7 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
         {createShorthand(Button, triggerButton, {
           defaultProps: () => ({
             className: dropdownSlotClassNames.triggerButton,
-            content,
+            content: { content, id: triggerButtonContentId },
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:576]: Just FYI: We don't merge `defaultProps` & user's input deeply, if someone will specify `content` i.e. `<Dropdown triggerButton={{ content: 'Foo }} />` this `id` will be gone. But now sure how it's important.

Comment (jurokapsiar) [reply packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:576]: I also thought, but if you look at line 553, it simplifies things:

```
const content = getSelectedItemAsString(value[0]);
```

so in this case content is always a string.

File: packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -562,7 +562,7 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
       },
       'aria-invalid': ariaInvalid,
       'aria-label': undefined,
-      ...(ariaLabelledby && { 'aria-labelledby': ariaLabelledby }),
+      'aria-labelledby': [ariaLabelledby, triggerButtonContentId].filter(l => !!l).join(' '),
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:565]:
```suggestion
      'aria-labelledby': [ariaLabelledby, triggerButtonContentId].filter(Boolean).join(' '),
```

_nit_

File: packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -573,37 +573,47 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
...
+            // It can be a shorthand
+            const resolvedContent = _.isPlainObject(predefinedProps.content)
+              ? (predefinedProps.content as {})
+              : predefinedProps.content
+              ? { children: predefinedProps.content }
+              : {};
+
+            return {
+              content:
+                predefinedProps.content === null ? null : { content, id: triggerButtonContentId, ...resolvedContent },
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx:592]:
```suggestion
                // If `null` is passed we should not render the slot
                predefinedProps.content === null ? null : { content, id: triggerButtonContentId, ...resolvedContent },
```

**Review by layershifter — APPROVED** [review]

**Pull request merged**